templates/common/page_start.html

```diff
@@ -19,7 +19,7 @@
     <title>Weasyl</title>
 
   <link rel="stylesheet" href="${CDNIFY('/static/fonts/museo500.css')}" />
-  <link rel="stylesheet" href="${CDNIFY('/static/public.css?' + SHA)}" />
+  <link rel="stylesheet" href="${resource_path('css/site.css')}" />
```

Comment:
gulp-rev adds the stylesheet’s hash to its filename, which `resource_path` then looks up. (And using the stylesheet’s hash rather than the commit’s means only stylesheet changes will cause it to be redownloaded.)